dali/test/python/test_external_source_parallel_pytorch.py

```diff
@@ -33,7 +32,6 @@ def __call__(self, sample_info):
 @raises(RuntimeError, "Error*starting Python worker threads for*parallel External Source*"
                       "Cannot fork*CUDA context already bound*"
                       "*start_py_workers*fork*spawn*")
-@with_setup(setup_function, teardown_function)
```

Comment:
Why remove that?

Reply:
This test doesn't create the pool at all as it fails on check for CUDA context, so it was unnecessary. On the other hand I wanted to make sure that tests with this decorator actually call capture_processes inside, so the teardown_function can make the check. In this case that assertion would error out.

Reply:
Understood.